docs: add a guide on importing data from a CSV file into PG (#6107)
hasura/graphql-engine: docs/graphql/core/guides/postgres/import-data-from-csv.rst

.. meta::
   :description: import data from csv into postgres
   :keywords: hasura, docs, postgres, import, data

.. _postgres_import_data_from_csv:

Import data from CSV into Postgres

> ===================================

Reviewer: Underline should match title length

.. contents:: Table of contents
  :backlinks: none
  :depth: 1
  :local:

Introduction
------------

You might have existing data stored in a CSV file that you need to import into your Postgres database. The following
guide will show how to do so.

Let's assume we have the following CSV file, which is named ``profile.csv``:

.. thumbnail:: /img/graphql/core/guides/sample-data-csv-file.png
   :alt: .csv data file
   :width: 500px

Step 1: Add a corresponding table to your PG database
-----------------------------------------------------

Let us create a table to match the data structure in your CSV file.

> Add a table as follows:

Reviewer: ```suggestion

```

Reviewer: The first sentence is enough

.. code-block:: bash

  profile (
    firstName TEXT, 
    lastName TEXT, 
    email VARCHAR
  )

Step 2: Connect to your Postgres database

> ------------------------------------------

Reviewer: Underline should match title length

> You can connect to your Postgres database using the ``psql`` command on the terminal like the following: 

Reviewer: ```suggestion
Connect to your Postgres database by using the ``psql`` command on the terminal: 
```
